ui/main/src/app/api/currentcard.api.ts

```
@@ -0,0 +1,156 @@
+                CurrentCardAPI.templateInterface.setTemplateRenderingComplete = listener;
+            },
+
+            registerFunctionToGetUserResponse: function (getUserResponse) {
+                CurrentCardAPI.templateInterface.getUserResponse = getUserResponse;
+            }
+        };
+
+        // prevent unwanted modifications from templates code
+        Object.freeze(opfab.currentCard);
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)